src/strongbox/core.clj

```diff
@@ -876,8 +877,11 @@
          tag-filter (if-not (empty? tag-set)
                       (fn [row]
                         (if-let [row-tag-set (set (:tag-list row))]
-                          ;; if the addon contains *some* of the selected tags, include it
-                          (some tag-set row-tag-set)))
+                          (if (-> filter-by :tag-membership (= "any of"))
```

Comment:
`case` this, there should be an error/noop if we get something other than specced values